fileio.c

```diff
@@ -0,0 +1,33 @@
+//Albert Zhang
+#include <stdio.h>
+#include <stdlib.h>
+int main () {
+	//writes into output file
+	//reads from input file
+	FILE *outFile = fopen("output.txt", "w");
+	FILE *inFile = fopen("input.txt", "r");
+	float arr[100];
+	int c;
+	for (c = 0; c <= 3; c++){
+		fscanf(inFile, "%f", &arr[c]);
+	}
+	for (c = 0; c <=3; c++) {
+		printf("%.2f \n", arr[c]);
+	}
+	//squares each from input, also averages
+	float square[3];
```

Comment:
Nice job, but just a few things:
1. your program only worked for four floats!
2. this line should be `float square[4];` - this gets rid of your warning! remember that you're declaring the size of the array~